Autos/score-and-back-up.txt

```diff
@@ -2,4 +2,4 @@ setCamera?pipeline=9
 aimTurret?timeMs=5000
 prepareFlywheel?timeMs=2000
 shoot?timeMs=5000
```

Comment:
We have new Autos on the DS that I forgot to grab. They use the new autoScore command that works better. We should grab them on Tuesday and add them to Github